charts/gatekeeper/values.yaml

```diff
@@ -17,7 +17,7 @@ validatingWebhookCustomRules: {}
 validatingWebhookURL: null
 enableDeleteOperations: false
 enableExternalData: true
-enableGeneratorResourceExpansion: false
+enableGeneratorResourceExpansion: true
```

Comment:
This change should be made https://github.com/open-policy-agent/gatekeeper/blob/master/cmd/build/helmify/static/values.yaml#L20 along with updating the default value here https://github.com/open-policy-agent/gatekeeper/blob/master/cmd/build/helmify/static/README.md

We don't directly modify charts/gatekeeper until we cut a release. FYI https://open-policy-agent.github.io/gatekeeper/website/docs/help#contributing-to-helm-chart

Reply:
Done. Thanks for the pointers.